### DISCUSSION TITLE
alt content on exemple (useless remark :p)

### DISCUSSION THREAD

**Discussion by vaggarath** [google/model-viewer#3217] [General]
I know it's silly but on https://modelviewer.dev/examples/scenegraph/#createTexturesExample the ALT is the one of the helmet (dunno why but everytime i block on it ^^'  )

**Comment by samaneh-kazemi** [comment]
Hi @vaggarath, thanks for the catch! I just sent a [PR](https://github.com/google/model-viewer/pull/3218) to fix it.
